CONTRIBUTING.md

```diff
@@ -145,3 +145,16 @@ Before submitting a pull request, please make sure that:
 ## Contributing documentation
 
 Please format and lint documentation with `task docs`.
+
+## Contributing code on Windows
```

Comment:
```suggestion
### Contributing code on Windows
```

And move the whole section before "Cloning the Repository". Otherwise, it seems strange to talk about contributing code again after talking about contributing documentation